dapp/src/components/dashboard/pages/MintBurn/index.jsx

```diff
@@ -117,12 +117,12 @@ const MintBurn = ({
         <TabPanel value={value} index={0}>
           <div className='mint-burn__balance'>
             <span className='title'>My Balance: </span>
-            <span className='amount'>{`(${capitalize(convertNetworkName(foreignNetwork))}) `}{balance ? formatWei(balance, 0, decimals) : 0}</span>
+            <span className='amount'>{`(${capitalize(convertNetworkName(foreignNetwork))}) `}{balance ? formatWei(balance, 2, decimals) : 0}</span>
             <small className='symbol'>{symbol}</small>
           </div>
           <MintBurnForm
             error={error}
-            balance={balance ? formatWei(balance, 0, decimals) : 0}
+            balance={balance ? formatWei(balance, 2, decimals) : 0}
```

Comment:
before that it rounded the numbers, right?

Reply:
Yes